glommio/src/task/raw.rs

```diff
@@ -215,64 +215,43 @@ where
     }
 
     /// Wakes a waker.
-    unsafe fn wake(ptr: *const ()) {
-        dbg_context!(ptr, "wake", {
-            let raw = Self::from_ptr(ptr);
-            if Self::thread_id() != Some(raw.my_id()) {
-                dbg_context!(ptr, "foreign", {
-                    let notifier = raw.notifier();
-                    notifier.queue_waker(Waker::from_raw(Self::clone_waker(ptr)));
-                });
-            } else {
-                let state = (*raw.header).state;
+    unsafe fn do_wake(ptr: *const ()) {
```

Comment:
this is unrelated to the channels work.

Usually I'd be okay with that, but we have had too many issues in this area. So please send this in a separate PR in case we need to bisect.

Reply:
Considering that the log is clean I was planning on rebasing. Taking that commit out wouldn't make much of a difference once rebased. Thoughts?

Reply:
different commits is good enough